Vagrantfile

```diff
@@ -13,7 +13,7 @@ Vagrant.configure(2) do |config|
   # Every Vagrant development environment requires a box. You can search for
   # boxes at https://atlas.hashicorp.com/search.
   config.vm.box = "hashicorp/precise32"
-  config.vm.provision :shell, path: "tutorails_bootstrap.sh"
+  # config.vm.provision :shell, path: "tutorails_bootstrap.sh"
   # config.vm.network :forwarded_port, guest: 3000, host: 3000
```

Comment:
we may need to uncomment this to tunnel the vm to the local dev environment. Thus, students could access the environment with `localhost:3000`